test-adapter/src/main/scala/org/scalajs/sbttestadapter/ScalaJSRunner.scala

```diff
@@ -135,24 +149,32 @@ final class ScalaJSRunner private[testadapter] (
 
   // Slave Management
 
-  private[testadapter] def getSlave(): ComJSRunner = {
+  /** Gets a slave for a [[ScalaJSTask]] which has already a started runner */
+  private[testadapter] def getRunnableSlave(): ComJSRunner = {
```

Comment:
The adjective "runnable" is misleading for this concept. When I deal with concurrency and processors, and I see "runnable", I think non-suspended. I'd suggest `getSlaveWithRunner` (and consistently renaming other usages of runnable slave).